db/models.py

```diff
@@ -1 +1,34 @@
 from django.db import models
+
+
+class Race(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    description = models.TextField(blank=True)
+
+
+class Skill(models.Model):
+    name = models.CharField(max_length=255)
+    bonus = models.CharField(
+        verbose_name="This field describes what"
+                     "kind of bonus players can get from it."
+                     "In other words, this is "
+                     "a description of the bonus.",
```

Comment:
The verbose_name for the 'bonus' field is quite long and could be simplified for clarity. Consider using a shorter description like 'Bonus description'.